### ISSUE TITLE
The saved model predicts different values ​​from the model trained in the session

### ISSUE THREAD

**Issue by VitoriaCarvalho** [alankbi/detecto#100]
Hey guys! I have a problem. Actually, maybe it's not a bug, but just something I didn't notice. I wonder if anyone had this problem.

So when I train the model and make the prediction, I have the bounding boxes and their respective labels. The problem is, if I save the model and load it again, the prediction returns different values ​​for the same input I used before.

I've already checked the `save()` and `load()` methods, in my opinion, they are right. I'm also taking care to apply the same `Compose()` that I used in training.

Has anyone gone through something like that? Thank you very much in advance :)

**Comment by VitoriaCarvalho** [comment]
I started to load the model this way and it worked:
```
loaded_model = core.Model(classes=classes)
loaded_model._model.load_state_dict(torch.load(path, map_location=loaded_model._device))
```
